### DISCUSSION TITLE
ModifiersClassNames Type

### DISCUSSION THREAD

**Discussion by elianarlivingston** [gpbl/react-day-picker#1955] [Support]
Hello! How are you? I wanted to inquire about the ModifiersClassNames type. Shouldn't the CustomModifiers also be partial? Thank you in advance!

<img width="788" alt="Captura de pantalla 2023-10-24 a la(s) 17 47 44" src="https://github.com/gpbl/react-day-picker/assets/59533564/1589aced-938d-4504-be58-d6d8313c66e7">

**Comment by gpbl** [comment]
Hi @elianarlivingston thanks for your question! I remember I was seeing something similar to do in the code  🤔 

Do you mean to update it to `Partial<Record<Modifier, string>> &
Partial<Record<InternalModifier, string>>;`?
